frontend/public/locales/en/common.json

```diff
@@ -254,7 +254,14 @@
   "website-verification-instruction": "Create a page at <1>{{webpage}}</1> containing the following token: <3>{{token}}</3> If the page already exists, add the token to it.",
   "website-verification-main-instruction": "Verify your right to use the app ID <1>{{id}}</1> by placing a token at a specific page on <3>{{domain}}</3>.",
   "login-provider-verification-main-instruction": "Verify your right to use the app ID <1>{{id}}</1> by logging into the account <3>@{{login_name}}</3> on <5>{{login_provider}}</5>.",
+  "login-provider-verification-main-instruction-organization": "Verify your right to use the app ID <1>{{id}}</1> by logging into an account with admin rights to the <3>@{{login_name}}</3> organization on <5>{{login_provider}}</5>.",
+  "login-provider-verification-enable-github-app": "To continue verification, Flathub needs permission to read your membership level in the <1>@{{organization}}</1> organization. You can grant this permission <3>in your GitHub settings</3>.",
   "login-provider-verification-switch-account": "You are currently logged in to <1>{{login_provider}}</1> as <3>@{{current_name}}</3>. Log in as <5>@{{required_name}}</5> to verify <7>{{appId}}</7> by login provider, or choose another method if one is available.",
+  "login-provider-verification-not-org-admin": "You are currently logged in to <1>{{login_provider}}</1> as <3>@{{current_name}}</3>. This account is not an admin of the <5>@{{organization}}</5> organization. To verify this app, you must request admin privileges in the organization or sign into an account that already has them.",
+  "login-provider-verification-not-org-member": "You are currently logged in to <1>{{login_provider}}</1> as <3>@{{current_name}}</3>. This account is not a member of the <5>@{{organization}}</5> organization. To verify this app, you must request admin privileges in the organization or sign into an account that already has them.",
+  "login-provider-verification-failed": "The verification requirements were still not met.",
```

Comment:
This sounds a bit passive aggressive, but I'm not a native english speaker

Reply:
Hmm, yeah. The idea was to give feedback when you click Try Again, since the status message above the button hasn't changed. But it can probably just say "The verification requirements were not met."